src/playground/sample_data_ingestion/jobs/refresh_job.py

```diff
@@ -0,0 +1,145 @@
+# opensearch-py==2.0.0
```

Comment:
Can we add unit tests for those two py files ? Thanks

Reply:
Done and with mock calls for all relevant tests.